charts/prow/generate_auth_k8s.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
```

Comment:
OK - I set them below the secret check because I want to have a custom error message.  I personally like the exists variable because, while maybe too verbose, it is more clear what the script is doing.  I did remove it and added comments in the script.